modules/ossm-config-control-plane-infrastructure-node-cli.adoc

```diff
@@ -3,12 +3,17 @@
 // * service_mesh/v2x/ossm-deployment-models.adoc
 
 :_content-type: PROCEDURE
-[id="ossm-config-control-plane-infrastructure-node_{context}"]
-= Configuring all {SMProductShortName} control plane components to run on infrastructure nodes
+[id="ossm-config-control-plane-infrastructure-node-cli_{context}"]
+= Configuring all control plane components to run on infrastructure nodes using the CLI
 
-This task should only be performed if all of the components deployed by the {SMProductShortName} control plane (including Istiod, Ingress Gateway, and Egress Gateway) along with optional elements (such as Prometheus, Grafana, and Distributed Tracing) are running on infrastructure nodes. 
+Perform this task if all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, will run on infrastructure nodes. 
```

Comment:
```suggestion
Perform this task if all of the components deployed by the {SMProductShortName} control plane will run on infrastructure nodes. These components include Istiod, Ingress Gateway, and Egress Gateway along with optional components, such as Prometheus, Grafana, and Distributed Tracing,
```
- This sentence is too information dense. Consider breaking into 2 sentences, maybe as I suggest here?
- Prometheus and Grafana are `components`, and not `elements`, I would say.

Reply:
Done

Reply:
done